<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newAnimalWindow</class>
 <widget class="QDialog" name="newAnimalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodawanie nowego zwierzaka</string>
  </property>
  <widget class="QGroupBox" name="groupBoxDown">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>330</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButtonCancel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Anuluj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConfirm">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Imię</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBoxNumber">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Numer boxu</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDisease">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Choroba?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDiseaseName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Nazwa choroby</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBreed">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Rasa</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFoundDate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Data YYYY-MM-DD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFoundStreet">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Lokalizacja</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHairColour">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Kolor sierści</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEyeColour">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Kolor oczu</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxUp">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>10</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButtonIsCat">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kot</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonIsDog">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pies</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>50</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButtonSexWoman">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Samica</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSexMan">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Samiec</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
